src/WebJobs.Script.Analyzers/AssemblyCache.cs

```diff
@@ -0,0 +1,190 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Hosting;
+using Microsoft.CodeAnalysis;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Azure.Functions.Analyzers
+{
+    // Only support extensions and WebJobs core.
```

Comment:
Could we add some summary comment to this type explaining what this class does?

Reply:
Removed for now, but when it comes back it'll be with comments. :)